zio-http-gen/src/main/scala/zio/http/gen/openapi/EndpointGen.scala

```diff
@@ -39,6 +39,12 @@ object EndpointGen {
         case None    => m - k
       }
   }
+
+  implicit class OptionCompanionCompatOps(o: Option.type) {
```

Comment:
You use this here only once. Please inline. Makes the code to complicated

Reply:
done